### ISSUE TITLE
Repository push error when used with Gitolite

### ISSUE THREAD

**Issue by SirIntellegence** [bozaro/git-as-svn#391]
Gitolite adds an update hook for itself in the git repository that returns an error when trying to do an svn check-in. See the second item at https://gitolite.com/gitolite/emergencies.html#common-errors. Removing/renaming the update hook so it doesn't get executed fixes git-as-svn while breaking Gitolite.

**Comment by slonopotamus** [comment]
Uh. I've got news for you. It's very likely that you're the very first person in the world that tries to set up git-as-svn on top of Gitolite. There are successful installations of git-as-svn + Gitea, git-as-svn + GitLab and git-as-svn + raw git repo. But it is very likely that Gitolite requires some special handling. Links that you gave says that Gitolite does not want its ssh access to be bypassed, but that's exactly how git-as-svn works! We will most likely need to figure out what things exactly Gitolite expects and pass it (in the context of the user who makes a commit through git-as-svn.

But first, would you be so kind to show your git-as-svn.conf (with all secred data redacted, obviously).

**Comment by SirIntellegence** [comment]
I have moved to Gita. It is a bit more than I need, but it works with git-as-svn. I am going to leave this open for now though in case someone else wants it?

**Comment by slonopotamus** [comment]
Yep, let's leave it open. Adding Gitolite support is is something that could be worth adding to git-as-svn.
